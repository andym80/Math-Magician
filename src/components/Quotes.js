function Quotes() {
  return (
    <div className="quotes">
      <p>
        Mathmatics is not about numbers,eauations, computations,
        or algorithms: it is about understanding. - William Paul Thurston
      </p>
    </div>
  )
}

export default Quotes;