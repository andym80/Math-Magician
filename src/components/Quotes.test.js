import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from  './Quotes';

it('renders to match the test', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });