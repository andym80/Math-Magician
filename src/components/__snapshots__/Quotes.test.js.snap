// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match the test 1`] = `
<div
  className="quotes"
>
  <p>
    Mathmatics is not about numbers,eauations, computations, or algorithms: it is about understanding. - William Paul Thurston
  </p>
</div>
`;
